packages/website/src/components/navigation/docs-nav-content.tsx

```diff
@@ -18,13 +17,14 @@ const NavContent = () => {
 
   return (
     <>
-      <NavHeader headerText="Docs" />
+      <NavHeader headerText="Docs" url="/docs" />
```

Comment:
this url will need to be changed once support for multiple docs folders is merged in

Reply:
yes